mkdocs.yml

```diff
@@ -175,8 +175,7 @@ markdown_extensions:
 
 # Plugins
 plugins:
-  - search:
-      separator: '[\s\-,:!?=\[\]()<>{}"/\\]+|(?!\b)(?=[A-Z][a-z])|\.(?!\d)|&[lg]t;'
+  - search
```

Comment:
```suggestion
  - search: '[\s\-,:!?=\[\]()<>{}"/\\]+|\.(?!\d)|&[lg]t;'
```

@TWiStErRob is this what you mean?

Reply:
careful with opening the yaml block
```suggestion
  - search:
      separator: '[\s\-,:!?=\[\]()<>{}"/\\]+|\.(?!\d)|&[lg]t;'
```

Reply:
If you commit this you should see the `(?!\b)(?=[A-Z][a-z])` part removed, along with one of the `|` (alternative operator). If you run this locally or on Codespaces you can try it out immediately.

Reply:
confirmed in codespaces:
![image](https://user-images.githubusercontent.com/2906988/213437339-ebd5cac3-864b-4474-85ee-935b535e778d.png)